applications/nrf5340_audio/src/modules/audio_usb.c

```diff
@@ -81,7 +84,8 @@ static void data_write(const struct device *dev)
 static void data_received(const struct device *dev, struct net_buf *buffer, size_t size)
 {
 	int ret;
-	void *data_in;
+	struct audio_data *audio_block = NULL;
+	struct net_buf *buf_in;
```

Comment:
Discuss: As a rule of thumb, all bufs which contain audio should IMO have the word "audio" in them.